Assets/Resources/Microgames/KaguyaMemory/Scripts/KaguyaMemory_BlackBarScript.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KaguyaMemory_BlackBarScript : MonoBehaviour {
+
+    [SerializeField]
+    private AudioClip clapSound;
+
+    public int myDirection = 1;                 //right is 1 and left is -1
+    public float stopPoint = 0;
+    private int timer1 = 0;
+    private int timer2 = 0;
+    public float movementSpeed = 4.0f;
+    private Rigidbody2D rb2d;
+    private bool exiting = false;
+
+	// Use this for initialization
+	void Start () {
+        rb2d = GetComponent<Rigidbody2D>();
+
+        //move based on direction given
+        movementSpeed *= myDirection;
+        rb2d.velocity = new Vector2(movementSpeed, 0f);
+    }
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
Same problem with using timer variables as in ItemScript and RNGDeciderScript. In this case, you might be better off using an animator for this.